ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -434,6 +489,14 @@ open class BottomCommandingController: UIViewController {
 
     private var bottomSheetController: BottomSheetController?
 
+    private var rootCommandingView: UIView? {
+        if isInSheetMode {
```

Comment:
Thoughts on using ternary notation here?
```
return isInSheetMode ? bottomSheetController?.view : bottomBarView
```

Reply:
I like it.